cmd/vicadmin/vicadm.go

```diff
@@ -96,8 +118,37 @@ var (
 )
 
 func init() {
-	trace.Logger.Level = log.DebugLevel
 	defer trace.End(trace.Begin(""))
+	trace.Logger.Level = log.DebugLevel
```

Comment:
vicadmin is spawned from vic-init, which (by necessity) runs as root.  vicadmin is a special case, since it runs http and is exposed to the outside world.  We can either put this code in vicadmin, or we can put it in vic-init as a special case and add the infrastructure to specify the user to run a specified command as.  Since this is a special case, I figured it would be easier to just put this code in vicadmin.